.github/workflows/tests.yml

```diff
@@ -0,0 +1,35 @@
+name: tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+  schedule:
```

Comment:
What's the intent behind running tests every Sunday? There doesn't immediately seem to be a use case that the other triggers don't cover?

Reply:
This is just something I like to do on all projects. It's less important while we don't currently have any unit tests, but it's helpful to know when something breaks because dependencies have released breaking changes. 

While you typically wouldn't get breaking changes from formatters/linters, it's not impossible. For example, black and ruff release style updates once a year, and this kind of scheduled run caught a lot of CI failures when those releases were made this past January.